.github/scripts/update_tools.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+""""
+Update scanner versions in the versions.yaml file.
+This script fetches the latest release versions of the specified scanners
+from their GitHub repositories and updates the version strings in the
+versions.yaml file while preserving the original file format.
+"""
+
+from pathlib import Path
+import os
+import sys
+import yaml
+import requests
+
+# Add the root directory to the Python path
+root_dir = Path(__file__).resolve().parents[2]
+
+def load_packages():
+    """Load package configurations from versions.yaml file."""
+    yaml_path = os.path.join(root_dir, "versions.yaml")
+    try:
+        with open(yaml_path, 'r', encoding='utf-8') as file:
+            data = yaml.safe_load(file)
```

Comment:
nit, but let's move to pathlib everywhere here
```suggestion
        data = yaml.safe_load(yaml_path.read_text(encoding="utf-8"))
```

(to me you do not need `os` import here and do not need `open`)